### PR TITLE
implement correct layout for jobseeker job alert view

### PR DESCRIPTION
**Pull request by alexbowen**
implement layout as per figma https://www.figma.com/file/wyEeHQ1U9aCXOJdzh8lF8a/Sprint-72?node-id=91%3A924

Screenshot 2021-02-08 at 20 11 08

### CODE REVIEW

File: app/views/jobseekers/subscriptions/index.html.slim [hunk 1]
```diff
@@ -18,14 +19,14 @@
...
+    = govuk_link_to(t(".button_create"), new_subscription_path(email: current_jobseeker.email, origin: jobseekers_subscriptions_path), button: true)
+
+.govuk-grid-row
   .govuk-grid-column-full
     .moj-filter-layout__content
       section
         - if @subscriptions.any?
-          .govuk-grid-row
-            .govuk-grid-column-full
-              = govuk_link_to(t(".button_create"), new_subscription_path(email: current_jobseeker.email, origin: jobseekers_subscriptions_path), button: true)
```
Comment (alexbowen) [inline app/views/jobseekers/subscriptions/index.html.slim:27]: i dont think this should only appear if you have existing subscriptions, should it?

**Comment by martin-bangoura** [comment]
Review app deployed to <https://teaching-vacancies-review-pr-2813.london.cloudapps.digital>

File: app/views/jobseekers/subscriptions/index.html.slim [hunk 2]
```diff
@@ -18,14 +19,15 @@
...
 
+- if @subscriptions.any?
+  .govuk-grid-column-one-third.create-alert__button
+    = govuk_link_to(t(".button_create"), new_subscription_path(email: current_jobseeker.email, origin: jobseekers_subscriptions_path), button: true)
+
+.govuk-grid-row
   .govuk-grid-column-full
     .moj-filter-layout__content
       section
         - if @subscriptions.any?
```
Comment (cpjmcquillan) [inline app/views/jobseekers/subscriptions/index.html.slim:30]: You can delete this line now there is line 22

File: app/frontend/src/styles/application/jobseekers/jobseeker.scss [hunk 3]
```diff
@@ -20,5 +20,17 @@
     .moj-filter-layout__content {
       overflow-x: hidden;
     }
+
+    .govuk-form-group {
```
Comment (cpjmcquillan) [inline app/frontend/src/styles/application/jobseekers/jobseeker.scss:24]: This feels quite broad. We can set classes on the specific form group by passing `form_group: { classes: "govuk-!-margin-0" }` to the form builder input for example

Comment (alexbowen) [reply app/frontend/src/styles/application/jobseekers/jobseeker.scss]: the style wasnt actually needed, the nesting of the markup was a little off

**Review by cpjmcquillan — APPROVED** [review]

**Pull request merged**